robo_a34.nv.json

```diff
@@ -0,0 +1,44 @@
+{
+	"_notes": ["Compiled by Horsepin."],
+	"_copyright": "Copyright (C) 2016 by Tom Collins <tom@tomlogic.com>",
+	"_license": "GNU Lesser General Public License v3.0",
+	"_endian": "big",
+	"_roms": ["robo_a34"],
+	"_fileformat": 0.1,
+	"_version": 0.1,
+	"last_game": [
+		{"start": "423", "encoding": "bcd", "length": 7, "packed": false},
+		{"start": "589", "encoding": "bcd", "length": 7, "packed": false},
+		{"start": "755", "encoding": "bcd", "length": 7, "packed": false},
+		{"start": "921", "encoding": "bcd", "length": 7, "packed": false}
+	],
+	"high_scores": [
+		{	"label": "Commander", "short_label": "GC",
+			"initials": {"start": "0x1C67", "encoding": "ch", "length": 3},
+			"score": {"start": "0x1C60", "encoding": "bcd", "length": 7,
+				"packed": false}
+		},
+		{	"label": "Commander", "short_label": "1st",
+			"initials": {"start": "0x1C73", "encoding": "ch", "length": 3},
+			"score": {"start": "0x1C6C", "encoding": "bcd", "length": 7,
+				"packed": false}
+		},
+		{	"label": "Commander", "short_label": "2nd",
+			"initials": {"start": "0x1C7F", "encoding": "ch", "length": 3},
+			"score": {"start": "0x1C78", "encoding": "bcd", "length": 7,
+				"packed": false}
+		},
+		{	"label": "Commander", "short_label": "3rd",
+			"initials": {"start": "0x1C8B", "encoding": "ch", "length": 3},
+			"score": {"start": "0x1C84", "encoding": "bcd", "length": 7,
+				"packed": false}
+		}		
+	],
+		"mode_champions": [
+		{	"label": "Jump Master", "short_label": "Jump Master",
+			"initials": {"start": "0x1C9F", "encoding": "ch", "length": 3},
+			"score": {"start": "0x1C98", "encoding": "int", "length": 1,
```

Comment:
If there's an easy way to get at least 10 jumps, or modify the .nv file to place a 0x10 there, it would be helpful to know if this is actually an "int" (so 0x10 would be 16 jumps) or packed BCD (where 0x10 would be 10 jumps).

Reply:
Yeah for sure is int.